helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixManager.java

```diff
@@ -117,7 +121,7 @@
   private final String _version;
   private int _reportLatency;
 
-  protected HelixZkClient _zkclient = null;
+  protected RealmAwareZkClient _zkclient = null;
```

Comment:
Nit: I know we don't have to change this, but initializing to null is redundant.